cmd/runtimetest/mount/README.md

```diff
@@ -0,0 +1 @@
+[code from](github.com/docker/docker/pkg/mount)
```

Comment:
Minor thing: If we're going to start pulling code from other repositories we should mention the license (I know it's the same as this project but it's a different project).